client/my-sites/plugins-wpcom/plugin-install-button.jsx

```diff
@@ -24,9 +26,15 @@ export const WpcomPluginInstallButton = props => {
 		siteId,
 		eligibilityData,
 		navigateTo,
-		initiateTransfer
+		initiateTransfer,
+		transferState
 	} = props;
 
+	// Don't show the button if the transfer has completed
```

Comment:
this is kind of self-explanatory in the code. what if we mentioned _why_ we don't want to show it

```js
// If this button is showing _and_ the transfer is complete then it
// means that this page was visible during the transfer process and
// we no longer want to expose the install button (it's already
// installed by this point). Instead, installs will be handled via the
// normal Jetpack plugins system.
```